packages/validators/src/schemas/v2023_09_28/pattern.ts

```diff
@@ -81,8 +81,8 @@ const ParameterDefinitionSchema = z.object({
   contentTypes: z.array(z.string()).min(1),
   passToNodes: z
     .array(PassToNodeSchema)
-    .max(1, 'At most one "passToNodes" element is allowed per parameter definition.')
-    .optional(), // we might change this to be empty array for native parameter definitions, that's why we don't use .length(1)
+    .max(1, 'At most one "passToNodes" element is allowed per parameter definition.'),
+  // we might change this to be empty array for native parameter definitions, that's why we don't use .length(1)
```

Comment:
nit: this comment can now be removed - also it's wrong 😄, we shouldn't use `length(1)` in any case.
```suggestion
```